components/dashboard/Dashboard.js

```diff
@@ -337,18 +360,25 @@ export default function Dashboard(props) {
               {...restProps}
             />
           </Box>
-          <Box className={classes.gridL}>
-            <Card variant="outlined">
-              <CardHeader
-                title="Confidence Level"
-                titleTypographyProps={{ component: 'h2', variant: 'h6' }}
-              />
-
-              <ActivityCategoryTable
-                allActivityLogEntries={allActivityLogEntries}
-                subsetActivityLogEntries={confidentActivityLogEntries}
-                label="Feeling Confident"
-              />
+          <Box className={classes.gridL} position="relative">
+            <RadioButtonUncheckedIcon className={classes.circleIcon} />
+            <TrackChangesIcon className={classes.trackIcon} fontSize="large" />
+            <Card className={classes.card} variant="outlined">
+              <CardContent>
+                <Typography variant="h6" gutterBottom>
+                  Employment Outlook
+                </Typography>
+                <Typography variant="body1" gutterBottom>
+                  Learn about the outlook in your area for the occupation that you want.
+                </Typography>
+              </CardContent>
+              <CardActions>
+                <NextLink href="/employment-outlook">
+                  <Button className={classes.exploreButton} fullWidth onClick={handleClick}>
+                    EXPLORE
+                  </Button>
+                </NextLink>
+              </CardActions>
             </Card>
```

Comment:
I was kind of expecting to see this in a new component called "EmploymentOutlook.js", but it seems Dashboard.js isn't very component heavy and also EmploymentOutlook isn't used anywhere else. So I guess this makes sense.